airflow/providers/amazon/aws/hooks/glue.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         self.role_name = iam_role_name
         self.s3_glue_logs = 'logs/glue-logs/'
         kwargs['client_type'] = 'glue'
-        super().__init__(*args, **kwargs)
+        super().__init__(region_name=region_name, *args, **kwargs)
```

Comment:
See https://github.com/apache/airflow/pull/14251#discussion_r577473549

Duplicate of #14251 -- which ever of you or @blcksrx update your PR first gets the merge 😁

Reply:
Thank you :)))